### PR TITLE
Items created from durathread are now 50% more resistant to consuming

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

Title.
This means that a moth can take, on average due to rounding, double the normal bites from an article of durathread clothing.
## Why It's Good For The Game

Minor moth and botanist buff. Means that moths will prefer eating durathread items where possible and will either work with botany or perhaps steal from others to get it
## Changelog
:cl:
add: NanoTrasen improved the quality of the local durathread strain; resulting in it now being twice as filling!
/:cl:

### CODE REVIEW

**Review by MrMelbert — APPROVED** [review]

**Comment by ju45he** [comment]
Just wait until shitcurity moffs start enslaving botanists for their durathread production

**Comment by Yttriums** [comment]
Botany nerf prs: 

- Makes all botany chems impure
- Gives all grown food a chance of causing disease
- Removes DNA analyzer
- Nerfs saltpeter and diethylamine (soon)

Botany buff prs:
- You may now eat 100% more durathread!

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -401,6 +401,7 @@
 
 	if(created)
 		created.setDir(builder.dir)
+		on_item_crafted(builder, created)
```
Comment (Ghommie) [inline code/game/objects/items/stacks/stack.dm:404]: All cool and dandy,  but this means only crafted items are affected by this, not ones that are pre-mapped or spawned through other means.

**Review by Ghommie — COMMENTED** [review]
I wish durathread, as well as cloth, would have material datums at some point.

File: code/game/objects/items/stacks/stack.dm (hunk 1 above)
Comment (ZephyrTFA) [reply code/game/objects/items/stacks/stack.dm:404]: I'm okay with this

**Pull request merged**